userdetails-plugin/grails-app/views/user/edit.gsp

```diff
@@ -44,7 +44,8 @@
             </g:hasErrors>
             <g:form method="post">
                 <g:hiddenField name="id" value="${userInstance?.userId}"/>
-                <g:hiddenField name="version" value="${userInstance?.version}"/>
+%{--                TODO: deal with optimistic locking--}%
```

Comment:
Removed this since the User Form object is not versioned.

<img width="428" alt="image" src="https://user-images.githubusercontent.com/72474143/216873013-049e9be2-0a01-4f8d-88d0-9ee2ab5b7822.png">

Reply:
Fixed.